Packs/VaronisSaaS/ReleaseNotes/1_0_6.md

```diff
@@ -0,0 +1,16 @@
+#### Incident Fields
+- update type to list for **Varonis SaaS Alert Status**
+- update type to list for **Varonis SaaS Close Reason**
+
+#### Mappers
+##### New: Varonis SaaS Incident - Outgoing Mapper
+- **dbotMirrorDirection** mapped on **mirror_direction**  
```

Comment:
@anas-yousef - is that common to add DBOT options here? assuming that the best way is just to mention that we have added an outgoing mapper and that's it.

Reply:
ok, no problem